src/main/java/com/launchacademy/reviews/controllers/PizzaStylesApiV1Controller.java

```diff
@@ -21,9 +24,32 @@ public PizzaStylesApiV1Controller(PizzaStyleService pizzaStyleService) {
   }
 
   @GetMapping
-  public Map<String, List<PizzaStyle>> getAllPizzaStyles() {
-    Map<String, List<PizzaStyle>> pizzaStylesMap = new HashMap<>();
+  public Map<String, Iterable<PizzaStyle>> getAllPizzaStyles() {
+    Map<String, Iterable<PizzaStyle>> pizzaStylesMap = new HashMap<>();
     pizzaStylesMap.put("pizzaStyles", pizzaStyleService.findAll());
     return pizzaStylesMap;
   }
+
+  @GetMapping("/{id}")
+  public Map<String, PizzaStyle> getById(@PathVariable Integer id){
+    Map<String, PizzaStyle> map = new HashMap<>();
+    Optional optional = pizzaStyleService.findById(id);
```

Comment:
Nice update here!

Reply:
I had to keep working and while waiting for a PR approval, so I branched off of this branch. Somehow those changes bled upstream. One has to assume I wasn't working in the branch I thought I was working in.